packages/cli/src/commands/execHandler.js

```diff
@@ -99,11 +99,18 @@ export const handler = async (args) => {
     ],
   })
 
-  try {
-    require.resolve(scriptPath)
-  } catch {
+  // check the script exists, and that there are no ambiguous conflicts like [foo.js, foo.ts]
+  const tgtScriptName = path.parse(scriptPath)?.name
```

Comment:
We prefer to not abbreviate words in variable names.
```suggestion
  const targetScriptName = path.parse(scriptPath)?.name
```